src/main/kotlin/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,48 @@
+package calculator
+
+private const val SEPARATOR = " "
+
+class StringCalculator {
```

Comment:
모든 메소드가 static한 메소드로 선언되어 있네요~
차라리 클래스 자체를 object로 선언해서 싱클턴 형태로 관리하는 것이 더 깔끔하지 않을까 싶어요

```suggestion
object StringCalculator {
```

https://kotlinlang.org/docs/object-declarations.html